components/home/ArcanaIntroVideo.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-overlay>
+    <iframe
+      src="https://www.youtube.com/embed/itP6iLLskro?autoplay=1"
+      title="What is Arcana? Mission | Benefits | Use Cases"
+      frameborder="0"
+      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+      allowfullscreen
+      class="absolute-center youtube-embed"
+    ></iframe>
+    <img
+      src="@/assets/icons/close-white.svg"
+      class="cursor-pointer position-absolute close-icon"
+      title="Close Video"
+      @click.stop="$emit('close')"
+    />
+  </v-overlay>
+</template>
+
+<script>
+export default {
+  name: 'ArcanaIntroVideo',
+  emits: ['close'],
+}
+</script>
+
+<style scoped lang="postcss">
+@import url('../lib/media-query-helper.css');
+
+.youtube-embed {
+  width: 64rem;
+  height: 36rem;
```

Comment:
Same width and height, but I find this easier to understand:

```suggestion
  width: 64rem;
  height: calc(64rem / 1.77);
```

Reply:
Sounds good. Resolved at https://github.com/arcana-network/website/pull/97/commits/79ab2fb2b7c95beebb50101aaab4dd86eca76084